indico/htdocs/js/indico/modules/categories/categories.js

```diff
@@ -175,4 +175,108 @@
             return $table.find(categoryRowSelector).has(checkboxSelector + ':checked');
         }
     };
+
+    global.setupCategoryEventList = function setupCategoryEventsList() {
+        enableIfChecked('#event-management', 'input[name=event_id]', '.js-enabled-if-checked');
+
+        function createCategoryNavigator(element, data) {
+            $('<div>').categorynavigator({
+                openInDialog: true,
+                selectLeafOnly: true,
+                onAction: function(category) {
+                    var confirmed = confirmPrompt($T.gettext('You are about to move some of the events to category ' +
```

Comment:
I don't think this works during message extraction for translation. AFAIK it needs to be a single string.